### PR TITLE
Please consider to merge this C# mono version

### PR DESCRIPTION
**Pull request by cuong0993**
Hi team,

I ported this demo to C#. Can we merge it?

### CODE REVIEW

**Comment by NathanLovato** [comment]
Thanks much! @Razoric480 may I ask you to check it and merge it, when you have the time?

File: mono-game/src/Actors/Enemy.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    [Export]
+    private int score = 100;
+    public override void _Ready()
+    {
+        SetPhysicsProcess(false);
+
+        _velocity.x = -speed.x;
+    }
+
+    public override void _PhysicsProcess(float delta)
```
Comment (Razoric480) [inline mono-game/src/Actors/Enemy.cs:16]: missing a call to `base._PhysicsProcess(delta);`, which causes enemies to ignore gravity.

**Review by Razoric480 — CHANGES_REQUESTED** [review]
Other than the missing call to the base physics process, looks to be working like the GDScript version.

File: mono-game/src/Actors/Enemy.cs (hunk 1 above)
Comment (cuong0993) [reply mono-game/src/Actors/Enemy.cs:16]: Hi, 
Thank for reviewing.
I fixed the defect.

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**